### PR TITLE
Improve documentation home page

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: doc/source/conf.py [hunk 1]
```diff
@@ -106,6 +106,8 @@ def setup(app):
     "sphinx.ext.intersphinx",
     "sphinx.ext.coverage",
     "sphinx_copybutton",
+    "sphinx_design",
```
Comment (Revathyvenugopal162) [inline doc/source/conf.py:109]: Add this as a dependency in toml file. the latest version is 0.5.0( FYI)

File: doc/source/Getting_started/Quickcode.rst [hunk 2]
```diff
@@ -0,0 +1,127 @@
+Qhick Code
```
Comment (Revathyvenugopal162) [inline doc/source/Getting_started/Quickcode.rst:1]:
```suggestion
Quick Code
```

File: doc/source/Getting_started/Quickcode.rst [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+On the `PyAEDT Issues <https://github.com/ansys/PyAEDT/issues>`_ page, you can
+create issues to report bugs and request new features. On the `PyAEDT Discussions
+<https://github.com/ansys/pyaedt/discussions>`_ page or the `Discussions <https://discuss.ansys.com/>`_
+page on the Ansys Developer portal, you can post questions, share ideas, and get community feedback.
+
+To reach the project support team, email `pyansys.core@ansys.com <pyansys.core@ansys.com>`_.
+
+
+Example Workflow
+================
```
Comment (Revathyvenugopal162) [inline doc/source/Getting_started/Quickcode.rst:35]:
```suggestion
================
```
We need to follow the rst headers. In that way we can avoid the error https://dev.docs.pyansys.com/how-to/documenting.html#indicating-rst-titles

File: doc/source/conf.py (hunk 1 above)
Comment (maxcapodi78) [reply doc/source/conf.py:109]: Done

File: doc/source/conf.py [hunk 4]
```diff
@@ -272,6 +274,24 @@ def setup(app):
...
+
+    Notes
+    -----
+    See https://jinja.palletsprojects.com/en/3.0.x/api/#jinja2.Environment
+    """
+    jinja_env.globals["project_name"] = project
+
+
+autoapi_prepare_jinja_env = prepare_jinja_env
+
```
[the remark this answers is not in the document]
Comment (maxcapodi78) [reply doc/source/conf.py:294]: I've commeted out

**Review by Samuelopez-ansys — APPROVED** [review]
This is great!

File: doc/source/Getting_started/index.rst [hunk 5]
```diff
@@ -1,8 +1,64 @@
-.. _ref_getting_started:
-
-===============
 Getting started
 ===============
+
+.. grid:: 2
```
[the remark this answers is not in the document]
Comment (maxcapodi78) [reply doc/source/Getting_started/index.rst:4]: Done Thanks!

**Review by Revathyvenugopal162 — APPROVED** [review]
LGTM, my only suggestion is check the size of image and also it will be better if the toctree on the left side of getting started and grid order are same. <when i checked locally it was not on same order.

**Pull request merged**